core/eolearn/tests/test_eodata.py

```diff
@@ -88,17 +91,19 @@ def test_vector_feature_types() -> None:
     "invalid_entry", [0, list(range(4)), tuple(range(5)), {}, set(), [1, 2, 4, 3, 4326, 3], "BBox"]
 )
 def test_bbox_feature_type(invalid_entry: Any) -> None:
```

Comment:
I find this usage of `invalid_entry` slightly annoying and would suggest to rename to `invalid_bbox`/`invalid_timestamps` across the tests, if you agree.

Reply:
sure, i have no attachments to this particular naming

Reply:
done